tutorials/vue/logos.md

```diff
@@ -0,0 +1,33 @@
+Here are the differnt versions of the Vue logo.
+
+## V
+
+### PNG
+
+<img src="https://vuejs.org/images/logo.png" alt="https://upload.wikimedia.org/wikipedia/commons/thumb/9/95/Vue.js_Logo_2.svg/1200px-Vue.js_Logo_2.svg.png" style="width:50%; height:50%"/>
```

Comment:
Download the images and add them to `/assets/images/vue`. External links get a bit messy because they can change or be removed.